src/test/java/org/opentripplanner/graph_builder/linking/LinkStopToPlatformTest.java

```diff
@@ -127,7 +127,7 @@ private AreaEdge createAreaEdge(
     LineString line = geometryFactory.createLineString(
       new Coordinate[] { v1.getCoordinate(), v2.getCoordinate() }
     );
-    I18NString name = new LocalizedString(nameString, new OSMWithTags());
+    I18NString name = LocalizedStringMapper.getInstance().map(nameString, new OSMWithTags());
```

Comment:
```suggestion
    I18NString name = new NonLocalizedString(nameString);
```

This way `LocalizedStringMapper` doesn't have to be public

Reply:
Thank you, I missed this one.